language/language_samples.rb

```diff
@@ -36,9 +38,11 @@ def sentiment_from_cloud_storage_file project_id:, storage_path:
 
   language  = Google::Cloud::Language.new project: project_id
   document  = language.document storage_path
-  sentiment = document.sentiment
-
-  puts "#{sentiment.score} (#{sentiment.magnitude})"
+  puts "Overall document sentiment: (#{document.sentiment.score})"
```

Comment:
You can reduce the number of chained method calls by creating a new variable sentiment = document.sentiment.